.github/workflows/inject-sidecar.yaml

```diff
@@ -0,0 +1,95 @@
+name: inject-sidecar
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - main
+
+jobs:
+  inject-istio:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: chrisdickinson/setup-yq@latest
+        with:
+          yq-version: v4.25.1
+      - name: get istio controlplane version
+        id: get-istio-version
+        run: |
+          ISTIO_VERSION=$(yq eval '. | select(.kind == "Deployment") | .spec.template.spec.containers[0].image | split(":") | .[1]' ./istio/operator/manifests.yaml)
+          echo "ISTIO_VERSION=$ISTIO_VERSION" >> $GITHUB_ENV
+      - name: get istioctl uri
+        id: get-istioctl
+        uses: istio/get-istioctl@v0.3
+        with:
+          version: ${{ env.ISTIO_VERSION }}
+      - name: Get Istioctl
+        run: |
+          istioctl version --remote=false
+
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.8"
+      - name: Inject Sidecar
+        id: inject
+        run: |
+          # hydrate isto manifest into configmap files
+          istioctl manifest generate \
+            -f istio/system/profile.yaml > manifests.yaml
```

Comment:
There is no longer a profile.yaml at that path.

Reply:
I think this command will need to be replaced with Helm template.  Let me see what I can do.